### DISCUSSION TITLE
alternative to mmds

### DISCUSSION THREAD

**Discussion by lperkov** [unikraft/unikraft#770] [General]
As there are many use-cases for MMDS-like communication between a VM (especially between the APP) and the out side world I'd love to hear your thoughts on either implementing (i) MMDS within unikraft or (ii) what are best practices to solve communication between the unikernelized app and the orchestrator?

**Comment by razvand** [comment]
@lperkov, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
